### PR TITLE
Feature/vijay common slider

### PR DESCRIPTION
**Pull request by mob-dev**
- Create & update common slider

┆Issue is synchronized with this [Jira Task](https://highvibe.atlassian.net/browse/HVN1-203) by [Unito](https://www.unito.io/learn-more)

### CODE REVIEW

File: mobile/lib/modules/audio_player/widgets/audio_player_slider.dart [hunk 1]
```diff
@@ -1,4 +1,6 @@
+import 'package:core/common/themes/app_colors.dart';
 import 'package:flutter/material.dart';
+import 'package:flutter_xlider/flutter_xlider.dart';
```
Comment (AlexVegner) [inline mobile/lib/modules/audio_player/widgets/audio_player_slider.dart:3]: Please flollow task https://github.com/HighVibe/hvn/issues/338
Move file to 
core/lib/widgets/sliders/common_slider.dart

Change class name to CommonSlider

**Review by AlexVegner — CHANGES_REQUESTED** [review]

File: mobile/lib/modules/audio_player/widgets/audio_player_slider.dart (hunk 1 above)
Comment (mob-dev) [reply mobile/lib/modules/audio_player/widgets/audio_player_slider.dart]: I have used the **flutter_xlider** package so I will not be able to change the import statement.

File: mobile/pubspec.yaml [hunk 2]
```diff
@@ -51,6 +51,8 @@ dependencies:
   video_player: ">=0.10.11+1 <2.0.0"
   virtual_keyboard: ^0.1.4
   web_socket_channel: ^1.1.0
+  web3dart: ^1.2.3
+  flutter_xlider: ^3.4.0
```
Comment (AlexVegner) [inline mobile/pubspec.yaml:55]: add it only to core or mobile + web 
mobile and web depends from code

File: web/pubspec.yaml [hunk 3]
```diff
@@ -73,6 +73,7 @@ dependencies:
   # ethereum wallet
   web3dart: ^1.2.3
   web_socket_channel: ^1.1.0
+  flutter_xlider: ^3.4.0
```
Comment (AlexVegner) [inline web/pubspec.yaml:76]: add it only to core or mobile + web
mobile and web depends from code

**Review by AlexVegner — CHANGES_REQUESTED** [review]

**Review by AlexVegner — APPROVED** [review]

**Pull request merged**